src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.content {
+  display: flex;
+  justify-content: center;
+  padding-top: 20px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-size;
+  padding: 0 $calendar-side-paddings;
+  gap: $calendar-gap;
+
+  @for $index from 1 through $calendar-max-days {
+    & :nth-child(#{$index}) {
+      display: none;
+      justify-content: center;
+      align-items: center;
+      width: $day-size;
+      height: $day-size;
+      background-color: #eee;
+      border: 1px solid black;
+      cursor: pointer;
+      transition-duration: 0.5s;
```

Always list styles that you apply transition for:
```suggestion
      transition: transform 0.5s, background-color 0.5s;
```